eng/scripts/automation/unified-pipeline-test.md

```diff
@@ -0,0 +1,186 @@
+# scenarios
```

Comment:
It would be potentially helpful if these could be separate files or perhaps embedded in another PS1 script that we can run for testing purposes. It can be a simple script that just creates the inputs calls your script and then compares the output with what you expect it to be.

Reply:
Hello @weshaggard So we will need to write script test as @benbp suggested. Right?

Reply:
Thank you, just to re-iterate please follow the Pester [unit test structure](https://github.com/Azure/azure-sdk-tools/blob/75f0ce338829ea4b6af65f4850022494811fdf80/doc/development/powershell.md#unit-testing).

Reply:
Hi @weshaggard  @benbp  I added tests under tests directories. Would you please take a look at it? Thanks

Reply:
Thanks for adding the tests and adding in the scenarios here. Can you also add some text in this MD that explains how to run the tests you created?